src/pages/workspaces/workspace/Data.js

```diff
@@ -593,6 +593,16 @@ const WorkspaceData = _.flow(
               retryFunction: loadEntityMetadata
             }, [
               _.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) && h(DataImportPlaceholder),
+              !_.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) && _.isEmpty(sortedEntityPairs) && div({
```

Comment:
Do you think it's worth pulling this out as a separate component (still within this file most likely) that takes the link text and callback as args? Then you would avoid the duplication of the 2 sections.

Reply:
@nawatts I actually meant both the styles and the link component-- the whole thing could be shared if you pass in the link text and the clickCallback.

Reply:
Eh, but then I guess you have to pass in 3 bits of information… maybe it isn't worth it.

Reply:
Nice, thanks!